docs/resiliency-checklist.md

```diff
@@ -83,12 +83,24 @@ It is preferable to scale horizontally than vertically. Use smaller Request and
 
 Make use of the [Quality of Service](https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html#quality-of-service-tiers) options available to ensure your pods are correctly resourced and treated appropriately by the scheduler.
 
-If your pod has its Requests and Limits set to `0` it will run BestEffort and simply use whatever spare capacity is available on its node. This can be good for things like batch jobs that want to go as fast as they can, but don't care about being slowed down from time to time. It will also be the first to be evicted from an overloaded node. Assigning `0` as a request or limit must be done through the CLI or directly in the manifest. 
+If your pod has its Requests and Limits set to `0` it will run BestEffort and simply use whatever spare capacity is available on its node. This can be good for things like batch jobs that want to go as fast as they can, but don't care about being slowed down from time to time. It will also be the first to be evicted from an overloaded node. Assigning `0` as a request or limit must be done through the CLI or directly in the manifest.
 
 If your pod has Limits higher than its Requests it will run Burstable and get at minimum the Requested amount and by burst up to its Limit depending on how busy the node is with other pods.
 
 If your pod has the Requests and Limits set to the same value then it will run Guaranteed QoS class. It will have preferential access to compute resources and will be the last to be evicted if a node should become overloaded. This is best for apps in the `prod` namespace as they need the best uptime.
 
+### A Well Behaved App
+
+It is important to ensure your app is well behaved and doesn't impede the cluster's operators work or impact the cluster's ability to heal.
```

Comment:
It prevents automation. Having the operators have to manually intervene should be avoided.

The upgrade process is now much more automated. We just click the go button and stand back. As the cluster gets bigger it will take longer to run and my run late into the evening. Having to manually intervene when it gets stuck will slow down the upgrade process.